src/Compilers/CSharp/Portable/Symbols/ConstraintsHelper.cs

```diff
@@ -803,7 +819,7 @@ private static bool HasDuplicateInterfaces(NamedTypeSymbol type, ConsList<TypeSy
             bool includeNullability,
             Compilation currentCompilation,
             ArrayBuilder<TypeParameterDiagnosticInfo> diagnosticsBuilder,
-            ArrayBuilder<TypeParameterDiagnosticInfo> warningsBuilderOpt,
+            ArrayBuilder<TypeParameterDiagnosticInfo> nullabilityBuilderOpt,
```

Comment:
>nullabilityBuilderOpt [](start = 54, length = 21)

Consider using the same name as in other methods.

Reply:
I converted every method I could find to the new name. Given this is only for nullability, as dictated in the doc comments, it flet like a more appropriate and readable name. Is there one I missed that still has the old name?

Reply:
Other methods use the name `nullabilityDiagnosticsBuilderOpt` while this method uses the name `nullabilityBuilderOpt`. Besides consistency, this name seems to imply we're building nullability rather than diagnostics.

---
In reply to: [257405054](https://github.com/dotnet/roslyn/pull/33336#discussion_r257405054) [](ancestors = 257405054)

Reply:
Ah I see what you mean now. 

---
In reply to: [257406316](https://github.com/dotnet/roslyn/pull/33336#discussion_r257406316) [](ancestors = 257406316,257405054)